com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipelineAsset.Migration.cs

```diff
@@ -173,7 +173,14 @@ enum Version
         Version m_Version = MigrationDescription.LastVersion<Version>();
         Version IVersionable<Version>.version { get => m_Version; set => m_Version = value; }
 
-        void OnEnable() => k_Migration.Migrate(this);
+        void OnEnable()
+        {
+            bool hasBeenMigrated = k_Migration.Migrate(this);
+#if UNITY_EDITOR
+            if (hasBeenMigrated)
+                UnityEditor.EditorUtility.SetDirty(this);
+#endif
```

Comment:
This should be directly done in Migrate method as it is valide for everything in Editor context when migrating.

Reply:
okay i was not sure here, i didnt want to enforce it for the whole system. but i ll move that code there!

Reply:
new version should contain the change.